apps/root/src/services/aggregatorService.ts

```diff
@@ -380,16 +380,20 @@ export default class AggregatorService {
     // eslint-disable-next-line no-plusplus
     for (let i = 0; i < logs.length; i++) {
       if (logs[i].address === byAddress) {
-        const parsedLog = decodeEventLog({
-          abi: contractAbi,
-          ...logs[i],
-        });
-        if (parsedLog.eventName === eventName) {
-          // @ts-expect-error typescript magic
-          if (!extraFilter || extraFilter(parsedLog)) {
+        try {
+          const parsedLog = decodeEventLog({
+            abi: contractAbi,
+            ...logs[i],
+          });
+          if (parsedLog.eventName === eventName) {
             // @ts-expect-error typescript magic
-            result.push(parsedLog);
+            if (!extraFilter || extraFilter(parsedLog)) {
+              // @ts-expect-error typescript magic
+              result.push(parsedLog);
+            }
           }
+        } catch {
+          // Skipping any event not included in our ABI
```

Comment:
we could 'console.error' the message just to debug it in instances where it fails and it shouldnt have